bootstrap.sh

```diff
@@ -354,12 +350,16 @@ get_shell_startup_script() {
 # Install Sentry CLI so that we can track errors that happen in this bootstrap script
 # This requires xcode CLI to be installed
 install_sentry_cli() {
+  log "Installing sentry-cli"
```

Comment:
I never quite grasped this `log`/`logk` dance. Am I doing it right?

Reply:
Yes it's just for a consistent terminal UI (was taken from github)